stackslib/src/net/rpc.rs

```diff
@@ -1223,6 +1225,42 @@ impl ConversationHttp {
         response.send(http, fd).map(|_| ())
     }
 
+    fn handle_validate_block_proposal(
+        metadata: HttpResponseMetadata,
+        block_proposal: BlockProposal,
+        options: &ConnectionOptions,
+    ) -> HttpResponseType {
+        // TODO: Reject if proposal is from invalid sender (allow localhost only?)
```

Comment:
Yes, you should do this before the PR merges.  It's `localhost` or bust.  If you're feeling ambitious, you can make it configurable in `ConnectionOptions`, but `localhost` will suffice.

Reply:
Pretty sure I did this already, I just need to remove the comment

Reply:
Already implemented, see https://github.com/jbencin/stacks-blockchain/blob/b1cd0afac08112bf6fdc79fc9fc50999710c1ee8/stackslib/src/net/rpc.rs#L3199